network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/image_text_mini.html

```diff
@@ -1,12 +1,15 @@
+{% extends "./base_streamfield_block.html" %}
 {% load wagtailcore_tags wagtailimages_tags %}
 
-<div class="my-5">
-  <div class="p-2 p-md-3 col-12 d-flex align-items-center image-text-mini-container">
-    <div class="image-text-mini-img-container mr-2 mr-md-3">
-      {% image value.image fill-100x100-c100 format-jpeg class="w-100" %}
-    </div>
-    <div class="image-text-mini-text">
-      {{ value.text|richtext }}
+{% block block_content  %}
+  <div class="my-5">
+    <div class="p-2 p-md-3 col-12 d-flex align-items-center image-text-mini-container">
```

Comment:
let's move that `my-5` into the `p-2 p-md-3 ...` and get rid out of the outer div, I don't know why it's there, but it's doing nothing the inner div can't already do, itself =)